### DISCUSSION TITLE
Termux-setup-storage on nix-on-droid

### DISCUSSION THREAD

**Discussion by RudraveerMandal** [nix-community/nix-on-droid#174] [Q&A]
I am trying to move my files from termux to nix-on-droid. I have moved them to the downloads directory on the phone. How do I get them to nix-on-droid?

**Comment by t184256** [comment]
https://github.com/t184256/nix-on-droid/wiki/Howto-grant-device-storage-access

**Reply by tennox** [reply]
Updated link:
https://github.com/nix-community/nix-on-droid/wiki/How-to-grant-device-storage-access
